plugins/Jobs.cpp

```diff
@@ -977,9 +971,10 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
 
         // If we are finishing a job that has child jobs, set its state to paused.
         if (SIEquals(requestVerb, "FinishJob") && _hasPendingChildJobs(db, jobID)) {
-            // Update the parent job to PAUSED
+            // Update the parent job to PAUSED. Also update its nextRun: in case it has a retryAfter, GetJobs set the nextRun too far in the future (to account for retryAfter), so set it to what it should
+            // be now that it is waiting on its children to complete.
             SINFO("Job has child jobs, PAUSING parent, QUEUING children");
-            if (!db.writeIdempotent("UPDATE jobs SET state='PAUSED' WHERE jobID=" + SQ(jobID) + ";")) {
+            if (!db.writeIdempotent("UPDATE jobs SET state='PAUSED', nextRun=" + SQ(lastRun) + " WHERE jobID=" + SQ(jobID) + ";")) {
```

Comment:
Why `lastRun`? Maybe the current time is better?

Reply:
I think `lastRun` is more correct because that's what GetJobs would do without retryAfter... I am not 100% sure though.

Reply:
ok, does not make much difference.